aptos-move/aptos-vm/src/move_vm_ext/session.rs

```diff
@@ -255,6 +265,42 @@ impl SessionOutput {
                 });
             my_changes.entries.extend(changes.entries.into_iter());
         }
+
+        // Squash aggregator changes.
```

Comment:
When is squash called?

Reply:
The only place where it's called is `encode_genesis_change_set`: https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/vm-genesis/src/lib.rs#L145

We squash two change sets since in genesis we:
1. Initialise things (validator sets, etc.) - session output 1.
2. Publish stdlib - session output 2.

In theory, this should never create deltas for this use case because these things only create rather write sets. But for completeness it makes sense to support squashing aggregator deltas?

Reply:
Ok this is fin - I added an assert and a proof-like comment